src/components/mobile-menu/mobile-menu.jsx

```diff
@@ -38,6 +39,17 @@ const MobileMenu = () => {
   return (
     <>
       <div className={classnames(styles.container, { [styles.openMenu]: showMenu })} >
+        {showTypeForm &&
+          <div className={styles.formContainer}>
+            <div onClick={() => setShowTypeForm(false)} className={styles.closeForm}><CloseIcon className={styles.closeFormIcon} /></div>
+            <iframe
+              src="https://form.typeform.com/to/c7G2RUWm?typeform-embed=popup-blank&typeform-source=blog.xmartlabs.com&typeform-medium=embed-sdk&embed-hide-footer=true&typeform-embed-id=rclfl" 
```

Comment:
Are you planning to turn this value into an environment variable??

Reply:
I don't know, because if the design team and the tester want to see the form, they would have to have the .env file, because it is not in the repo (I understand that I should not upload it).

Reply:
They shouldn't need to access the repo anyway. This link can be public for the team in slack or notion

Reply:
Yes, but for the deploy they need the link, and if the .env file is not in the repo, they can't access the form because of the environment variable.

Reply:
let's leave it this way for now. Netlify should have a way to inject env vars from somewhere.